document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/service/impl/AccountsServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.companieshouse.document.generator.accounts.service.impl;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.model.accounts.Accounts;
+import uk.gov.companieshouse.document.generator.accounts.data.accounts.AccountsManager;
+import uk.gov.companieshouse.document.generator.accounts.service.AccountsService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+@Service
+public class AccountsServiceImpl implements AccountsService {
+
+    private static final Logger LOG = LoggerFactory.getLogger("document-generator-accounts");
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Accounts getAccounts(String resource) {
+        LOG.info("Getting accounts data from abridged-accounts-api");
+
+        ResponseEntity<Accounts> accounts = AccountsManager.getAccounts(resource);
+
+        if (accounts.getStatusCode() != HttpStatus.OK) {
+            LOG.error("Failed to retrieve data from API: " + resource);
+            return null;
+        }
+        LOG.trace("Accounts data retrieved successfully");
```

Comment:
Is this log going to benefit support or cause noise in the logs (it has no reference to which account was retrieved and cannot be linked to the request)?

Reply:
I believe it is handy to know that it has been retrieved successfully. Additionally, it isn't retrieving a certain type of accounts - just the base account resource.

Reply:
Sorry if I was unclear. By "no reference to which account" I meant no identifier (e.g. account ID, account type, transaction ID) .

Reply:
Ahhh I see, I am going to tweak the logging to make it more robust in following PRs but just to get something in I have added the resource.